src/js/otus/hooks.ts

```diff
@@ -0,0 +1,35 @@
+import { useFetchOTU } from "@otus/queries";
+import { OTUSequence } from "@otus/types";
+import { indexOf, map, sortBy } from "lodash-es/lodash";
+
+/**
+ * A hook for sorting the sequences for the active isolate
+ *
+ * @param otuId - The otu which the isolate belongs to
+ * @param sequences - The sequences in the active isolate
+ */
+export default function useGetSequences(otuId: string, sequences: OTUSequence) {
+    const { data, isLoading } = useFetchOTU(otuId);
```

Comment:
Why call `useFetchOtu`? looks to me like the data returned from it isn't getting used